### DISCUSSION TITLE
Issue: documentation, Mermaid (markdown) sequenceDiagram `create participant...` does not work

### DISCUSSION THREAD

**Discussion by papadeltasierra** [community/community#76561] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I cannot figure out how to raise an issue against GitHub itself so if it is possible, please tell me now. But for now...
The following valid Mermaid Markdown syntax does not display in the GitHub preview panels - instead a meaningless error is displayed.
```
sequenceDiagram;
    participant Alice
    create participant Bob
    Alice->>Bob: Hello
```
If you remove the `create` before `participant Bob`, it works fine.  Other Mermaid renderers (e.g. VSCode add on. Mermaid live etc) correctly render this code.

**Comment by mlhamatms** [comment]
So, We have the same problem with `erDiagram`. Works in live and VS code but not in GH.

**Reply by mlhamatms** [reply]
I figured it out I think ... I do not believe that GH is up to 10.5.x yet is it? For what we are doing we wanted to use labels so we need 10.5.x to get that in erDiagram type.
